.agents/skills/typescript-expert/scripts/ts_diagnostic.py

```
@@ -0,0 +1,225 @@
+        sample = run_cmd(f"grep -rn ': any' --include='*.ts' --include='*.tsx' {src_path} 2>/dev/null | head -5")
+        if sample:
+            print(sample)
+    else:
+        print("  ✅ No explicit 'any' types found")
+
+def check_type_assertions(project_root: str = "."):
+    """Check for type assertions."""
+    print("\n⚠️ Type Assertions (as):")
+    print("-" * 40)
+    
+    src_path = os.path.join(project_root, "src") if project_root != "." else "src/"
```

Ruff (0.15.1)

[error] Undefined name `os`

(F821)